src/app/App.tsx

```diff
@@ -26,13 +27,25 @@ const LibraryStackScreen = () => {
     headerBackground: () => <Image style={styles.imageHeader} source={require('./assets/bc_nav_bar.png')} />,
     headerStyle: styles.headerStyle,
     cardStyle: styles.headerCard,
-    headerTitleStyle: styles.headerTitle,
-    headerLeft: () => <LogoutHeader />
+    headerTitleStyle: styles.headerTitle
   };
   return (
     <LibraryStack.Navigator initialRouteName="Library" screenOptions={headerOptions}>
-      <LibraryStack.Screen name="Library" component={Library} />
+      <LibraryStack.Screen
+        name="Library"
+        component={Library}
+        options={{ headerLeft: () => <LogoutHeader />, headerRight: () => <SearchHeader extended={false} /> }}
+      />
       <LibraryStack.Screen name="BookDetail" component={BookDetail} />
+      <LibraryStack.Screen
+        name="LibrarySearch"
+        component={Library}
+        options={{
+          headerTitle: () => <SearchHeader extended={true} />,
```

Comment:
you don't need to set the value to true, when you add it, it is automatically setted to true

Reply:
Yes! Changed.